react-native-purchases-ui/src/index.tsx

```diff
@@ -77,12 +82,30 @@ type FullScreenPaywallViewProps = {
   style?: StyleProp<ViewStyle>;
   children?: ReactNode;
   options?: FullScreenPaywallViewOptions;
+  onPurchaseCompleted?: ({
+                           customerInfo,
+                           storeTransaction
+                         }: { customerInfo: CustomerInfo, storeTransaction: PurchasesStoreTransaction }) => void;
```

Comment:
Hmm I wonder if we should create types for these... But it's probably ok for now.